xtask/codegen/html.ungram

```diff
@@ -68,11 +69,30 @@ HtmlElementList = AnyHtmlElement*
 AnyHtmlElement =
 	HtmlSelfClosingElement
 	| HtmlElement
-	| HtmlContent
+	| AnyHtmlContent
 	| HtmlComment
 	| HtmlCdataSection
 	| HtmlBogusElement
 
+AnyHtmlContent =
+	HtmlContent
+	| AnyHtmlTextExpression
+
+AnyHtmlTextExpression =
+	HtmlVueTextExpression
+	| HtmlSvelteTextExpression
+
+
+// {{ variable }}
+HtmlVueTextExpression =
+	'{{'
+	expression: 'html_literal'
+	'}}'
+
+HtmlSvelteTextExpression =
+	'{'
+	expression: 'html_literal'
+	'}'
```

Comment:
With regards to naming things -- I think it makes sense to omit the `Html` prefix when the node is a part of some other super language.

So in these cases, the name would be `VueTextExpression` and `SvelteTextExpression`. This would be consistent with how we name `Js` vs `Jsx` vs `Ts` nodes, for example.